### PR TITLE
[NUI] Add some APIs to make Accessibility enabled / disabled

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- Added EnableAccessibility and IsEnabled to AccessibilityManager
- dali-csharp-binder : https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/248417/

### API Changes ###
- N/A

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Interop/Interop.AccessibilityManager.cs [hunk 1]
```diff
@@ -89,6 +89,12 @@ internal static partial class AccessibilityManager
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_AccessibilityManager_GetReadPosition")]
             public static extern global::System.IntPtr AccessibilityManager_GetReadPosition(global::System.Runtime.InteropServices.HandleRef jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_AccessibilityManager_EnableAccessibility")]
+            public static extern void AccessibilityManager_EnableAccessibility(global::System.Runtime.InteropServices.HandleRef jarg1, bool jarg2);
+
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.AccessibilityManager.cs:94]: please remove "\_" in NUI API naming. I think AccessiblityManager is repeated.
so how about this ?
AccessibilityManager_EnableAccessibility => EnableAccessibility
I can see other legacy code uses "_" here, so please change these namimgs for next time.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Comment by dongsug-song** [comment]
@Seoyeon2Kim 
Hello, please see my comment and do merge it for yourself this time~
thank you.

File: src/Tizen.NUI/src/internal/Interop/Interop.AccessibilityManager.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/internal/Interop/Interop.AccessibilityManager.cs:94]: Thank you for your review.
I updated it to remove the duplicated prefix. :)

**Pull request merged**